src/utils/withAltContext.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 export default function withAltContext(flux) {
-  return function (Component) {
+  return function(Component) {
```

Comment:
Can we revert these? My take on it is that function is a keyword similar to if, while, etc and should be treated as such in regards to spacing.